convert/convert.go

```diff
@@ -318,10 +353,17 @@ func (rr *TsdbRowReader) ReadRows(buf []parquet.Row) (int, error) {
 		buf[i] = rr.rowBuilder.AppendRow(buf[i][:0])
 		i++
 	}
+
 	rr.totalRead += int64(i)
+
+	if rr.ctx.Err() != nil {
+		return 0, rr.ctx.Err()
+	}
+
 	if i < len(buf) {
```

Comment:
just noticed, this has a bug where when we skip series because all chunks are empty we might run into this ( i.e. when we convert a 2d block and second day has chunks for some series but not first day.

Reply:
nice catch!!! :D 

thanks!!

Reply:
i couldnt write a failin test - all good